### DISCUSSION TITLE
Density model from gravity

### DISCUSSION THREAD

**Discussion by gnssgrid** [fatiando/community#148] [General]
Hi
Is it possible to compute 2D or 3D density model from grided complete Bouguer anomalies?
Regards

**Comment by santisoler** [comment]
Hi @gnssgrid. Thanks for opening the question. The process of obtaining a model of the subsurface from the observed data is known as an [inversion problem](https://en.wikipedia.org/wiki/Inverse_problem).

Solving an inverse problem is usually not a straight forward process mainly because it's an ill-posed problem: non-unique and non-stable solutions. But there are open-source tools that allows you to run inversions. To name a few: [SimPEG](https://simpeg.xyz) and [PyGIMLi](https://www.pygimli.org/).

Also, check out https://github.com/softwareunderground/awesome-open-geoscience for a comprehensive list of open source tools for geophysics.

**Reply by gnssgrid** [reply]
Hi @santisoler 
Thanks for information. I will check SimPEG and PyGIMLI
